src/pages/supplies_new.js

```diff
@@ -41,7 +41,7 @@ function NewSupplyRequest({ backend, history }) {
             />
           }
         >
-          <SupplyConfirmation onEdit={() => setSent(false)} name="test" />
+          <SupplyConfirmation onEdit={() => setSent(false)} />
```

Comment:
Was this supposed to make it into this PR?

Reply:
I only ask because I rewrote this piece a bit in another PR.

Reply:
Oh do we need that `name` attrib? I thought I checked the history on that line and saw it was me, assumed it was a leftover value...

Reply:
We don't! I removed it in another PR along with the `onEdit`. That component and page are undergoing some changes.

Reply:
In another *Pending PR.

Reply:
https://github.com/helpsupply/helpsupply/pull/156/files#diff-5af4298448086bc0d1561001750332e3

Reply:
Ahh right right, I'll rebase to master and accept changes here. Should be easy.